README.md

```diff
@@ -17,7 +17,7 @@
 This package models Salesforce Marketing Cloud data from [Fivetran's connector](https://fivetran.com/docs/connectors/applications/salesforce-marketing-cloud). It uses data in the format described by [this ERD](https://fivetran.com/docs/connectors/applications/salesforce-marketing-cloud#schemainformation).
 
 The main focus of the package is to transform the core object tables into analytics-ready models:
-<!--section="salesforce_marketing_cloud_model"-->
+<!--section="salesforce_marketing_cloud_transformation_model"-->
```

Comment:
Actually, perhaps should this line be deleted? I see we have a duplicate on line 28 , which is the more appropriate position for the tag I think, so could you update the section tag there?

Reply:
Oh, definitely, I am also confused :D